cluster-autoscaler/processors/emptycluster/empty_cluster_processor.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package emptycluster
+
+import (
+	"time"
+
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/autoscaler/cluster-autoscaler/clusterstate/utils"
+	"k8s.io/autoscaler/cluster-autoscaler/context"
+	"k8s.io/autoscaler/cluster-autoscaler/metrics"
+	"k8s.io/klog/v2"
+)
+
+// How long should Cluster Autoscaler wait for nodes to become ready after start.
+const nodesNotReadyAfterStartTimeout = 10 * time.Minute
+
+// EmptyClusterProcessor is the interface for empty cluster processor
+type EmptyClusterProcessor interface {
+	ActOnCluster(context *context.AutoscalingContext, allNodes []*apiv1.Node, readyNodes []*apiv1.Node, currentTime time.Time) bool
+	CleanUp()
+}
+
+// DefaultEmptyClusterProcessor is the struct for EmptyCluster Process
```

Comment:
Similar to above - I don't need comment to understand that this is an implementation of the interface. What I'd like to learn from the comment is what this implementation does. I think https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/processors/customresources/gpu_processor.go#L32 is a good example for how those comments can provide an idea for what the implementation does.